src/main/scala/firrtl/stage/FirrtlAnnotations.scala

```diff
@@ -221,15 +222,16 @@ object RunFirrtlTransformAnnotation extends HasShellOptions {
 
   private[firrtl] def stringToEmitter(a: String): RunFirrtlTransformAnnotation = {
     val emitter = a match {
-      case "none"     => new ChirrtlEmitter
-      case "mhigh"    => new MinimumHighFirrtlEmitter
-      case "high"     => new HighFirrtlEmitter
-      case "low"      => new LowFirrtlEmitter
-      case "middle"   => new MiddleFirrtlEmitter
-      case "verilog"  => new VerilogEmitter
-      case "mverilog" => new MinimumVerilogEmitter
-      case "sverilog" => new SystemVerilogEmitter
-      case _          => throw new OptionsException(s"Unknown compiler name '$a'! (Did you misspell it?)")
+      case "none"               => new ChirrtlEmitter
+      case "mhigh"              => new MinimumHighFirrtlEmitter
+      case "high"               => new HighFirrtlEmitter
+      case "low"                => new LowFirrtlEmitter
+      case "middle"             => new MiddleFirrtlEmitter
+      case "verilog"            => new VerilogEmitter
+      case "mverilog"           => new MinimumVerilogEmitter
+      case "sverilog"           => new SystemVerilogEmitter
+      case "experimental-rtlil" => new RtlilEmitter
+      case _                    => throw new OptionsException(s"Unknown compiler name '$a'! (Did you misspell it?)")
```

Comment:
Please undo your changes to this file. This code is for the (soon to be) deprecated `-X` option. New emitters should only use the `-E` option.